app/scripts/controllers/about.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+angular.module('openhimConsoleApp')
+  .controller('AboutCtrl', function ($rootScope, $scope, Api, Alerting, Notify, config) {
+    
+    var success = function(result) {
+      $scope.aboutInfo = result;
+      Notify.notify('aboutInfoReceived');
```

Comment:
Dont think there is a need to use Notify here. You can call the buildAboutInfoObject() function when the Api call is successful